homeassistant/components/miele/sensor.py

```diff
@@ -102,12 +105,47 @@ def _get_coffee_profile(value: MieleDevice) -> str | None:
     return None
 
 
+def _convert_start_timestamp(
+    elapsed_time_list: list[int], start_time_list: list[int]
+) -> datetime | None:
+    """Convert raw values representing time into start timestamp."""
+    now = dt_util.now()
```

Comment:
Why are we using local time here instead of utc? We generally want to use utc, and that's also how the sensor's state will be rendered in the end. I suggest to call `dt_util.utcnow()` instead.